src/Microsoft.ML.TimeSeries/RootCauseAnalyzer.cs

```diff
@@ -40,7 +40,7 @@ private RootCause AnalyzeOneLayer(RootCauseLocalizationInput src)
             dst.Items = new List<RootCauseItem>();
 
             DimensionInfo dimensionInfo = SeparateDimension(src.AnomalyDimension, src.AggregateSymbol);
-            Tuple<PointTree, PointTree, Dictionary<Dictionary<string, object>, TimeSeriesPoint>> pointInfo = GetPointsInfo(src, dimensionInfo);
+            Tuple<PointTree, PointTree, Dictionary<Dictionary<string, Object>, TimeSeriesPoint>> pointInfo = GetPointsInfo(src, dimensionInfo);
```

Comment:
do we have to change to `Object` from `object`? can we add a test to cover this case? especially using the long/int/double/decimal rather than string

Reply:
> do we have to change to Object from object? 

We should be using the C# keyword `object` here, and not `Object`. This repo's [coding style](https://github.com/dotnet/machinelearning/blob/master/docs/project-docs/contributing.md#coding-style-changes) follows the same [coding style from dotnet/runtime](https://github.com/dotnet/runtime/blob/master/docs/coding-guidelines/coding-style.md). See rule 11:

> We use language keywords instead of BCL types (e.g. int, string, float instead of Int32, String, Single, etc) for both type references as well as method calls (e.g. int.Parse instead of Int32.Parse).

Reply:
I find that we use Object most of time in this root cause localization extension, such as the definition of the input and output. I just want to make the same coding style. I can discard this change for this PR, but there would be two different coding style using object/Object in current implementation. @eerhardt Or do we need to replace all Object by object reversely? or just stay the same as the current version.

Reply:
revert the coding style change